test-cases/gemini/gemini-basic-3h.yaml

```diff
@@ -10,9 +10,9 @@ store_results_in_elasticsearch: False
 # gemini
 # cmd: gemini -d -n [NUM_OF_TEST_ITERATIONS] -c [NUM_OF_THREADS] -p [NUM_OF_PARTITION_KEYS_PER_THREAD] -m mixed -f
 # the below cmd runs about 3 hours
-gemini_cmd: "gemini -d -n 500000 -c 20 -p 100 -m mixed -f"
-gemini_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/gemini-latest'
-gemini_static_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
+gemini_cmd: "gemini -d --duration 120s -c 20 -p 100 -m mixed -f"
+gemini_version: '0.9.2'
```

Comment:
According to this parameter we choose the gemini version to download Or is it just for annotation?
If the former, maybe it worth to link to latest or something so we won't need to manually replace the version every time.

Reply:
it downloads gemini by version

Reply:
let's keep version to track which version ran

Reply:
We can track the version we run by checking it and storing it in ealsticsearch.
How do you suggest to keep moving forward with Gemini versions?
We can ask them to transfer us the gemini version from their upstream job maybe, and replace it here if possible.

Reply:
We can track by email and change manually when we want. But it should be a controlled process. Imagine that they released new version with a bug and you use always latest gemini job will fail.
![image](https://user-images.githubusercontent.com/5269241/57076841-37c4ae00-6cf3-11e9-8f87-ff636c8f0b9a.png)

Reply:
As we discussed, we don't want to maintain it manually and we're ok if we find issues with Gemini that may cause failures.
We approves this for now and open a new task for tracking "latest" gemini and running with latest option.